src/redux/charts.js

```diff
@@ -40,18 +41,36 @@ export const chartsUpdateChartType = (chartType, dpi) => dispatch =>
     type: CHARTS_UPDATE_CHART_TYPE,
   });
 
-export const assetChartsReceived = message => (dispatch, getState) => {
+const formatChartData = chart => {
```

Comment:
I moved this from `src/hooks/charts/useChartData.js` since this is the only place where we use this